### PR TITLE
added crucher stub files

### PR DESCRIPTION
**Pull request by vijayakarumudi**
Closes AgileVentures/MetPlus_tracker#501

### CODE REVIEW

**Comment by vijayakarumudi** [comment]
PR for review

File: spec/mailers/previews/company_mailer_preview.rb [hunk 1]
```diff
@@ -1,3 +1,6 @@
+include WebMock::API
```
Comment (patmbolger) [inline spec/mailers/previews/company_mailer_preview.rb]: I get this error when I run this:

screen shot 2017-01-10 at 11 00 05 am

To fix this, out this line in before the first line in the file:
```
require 'webmock'
```

File: spec/mailers/previews/company_mailer_preview.rb [hunk 2]
```diff
@@ -22,10 +25,12 @@ def registration_denied
...
   def application_received
+    WebMock.enable!
+    stub_cruncher_authenticate
+    stub_cruncher_file_download('./spec/fixtures/files/Janitor-Resume.doc')
     company = Company.last
     job_application = JobApplication.last
-    CompanyMailer.application_received(company, job_application,
-        "#{Rails.root}/spec/fixtures/files/Janitor-Resume.doc")
+    CompanyMailer.application_received(company, job_application, 1)
   end
```
Comment (patmbolger) [inline spec/mailers/previews/company_mailer_preview.rb]: I see this when I run this:
screen shot 2017-01-10 at 11 00 05 am

This is tricky as the data in a dev DB may be different from user to user.  And, we do not want to create objects in the dev DB.  This code worked for me:
```
def application_received
    WebMock.enable!
    stub_cruncher_authenticate
    stub_cruncher_file_download('./spec/fixtures/files/Janitor-Resume.doc')

    job_application = nil
    JobApplication.all.each do |ja|
      next if ja.job_seeker.resumes.empty?
      job_application = ja
      break
    end

    raise 'Cannot find application with resume' unless job_application

    CompanyMailer.application_received(job_application.job.company,
                                       job_application,
                                       job_application.job_seeker.resumes[0].id)
end
```

File: spec/mailers/previews/company_mailer_preview.rb (hunk 1 above)
Comment (vijayakarumudi) [inline spec/mailers/previews/company_mailer_preview.rb]: Before I rebased code it is working. We fixed this problem in pairprograming.

**Comment by vijayakarumudi** [comment]
updated PR.

**Pull request merged**